packages/base/components/Image/ImageWithAttribution.vue

```diff
@@ -52,8 +52,15 @@ const props = defineProps({
 </script>
 <template>
   <figure>
+    <ImageLottiePlayer
+      v-if="
+        props.src &&
+        ['application/json', 'application/zip'].includes(props.contentType)
```

Comment:
This seems rather broad, matching any json or zip file...

How about we instead convert the lottie file to a dotlottie file, which makes it much smaller, then here look not only at the content type (zip) but also the file extension (.lottie)?